frontend/src/components/eMIB/TestInstructions.jsx

```diff
@@ -47,23 +48,33 @@ class TestInstructions extends Component {
             <p>{LOCALIZE.emibTest.howToPage.testInstructions.step1Section.part3.para3}</p>
           </div>
           <div>
-            <p className="font-weight-bold">
+            <h3 className="emib-subsection-titles">
               {LOCALIZE.emibTest.howToPage.testInstructions.step2Section.title}
-            </p>
+            </h3>
             <p>{LOCALIZE.emibTest.howToPage.testInstructions.step2Section.description}</p>
             <p className="font-weight-bold">
               {LOCALIZE.emibTest.howToPage.testInstructions.step2Section.example.title}
             </p>
             <p className="font-weight-bold underline">
               {LOCALIZE.emibTest.howToPage.testInstructions.step2Section.example.part1Title}
             </p>
-            {/* TODO: (fnormand01) Find a way to put the French image when we click on 'French' button */}
             <p>
-              <img
-                src={original_email_emib_sample_test_example_en}
-                alt={LOCALIZE.emibTest.howToPage.testInstructions.step2Section.example.part1Title}
-                className="test-instruction-image"
-              />
+              {LOCALIZE.emibTest.howToPage.testInstructions.step2Section.example
```

Comment:
This is definitely getting there, but not ideal. Once we have redux set up, we'll use that to toggle images. So that it's not dependent on a string change.

Reply:
Yeah I know. This is only temporary. We really need _redux_ for this type of thing.